pkgs/development/python-modules/spacy/default.nix

```diff
@@ -46,6 +46,12 @@ buildPythonPackage rec {
     hash = "sha256-Szfr0lraQFmw3J4Ik+cN3l34NIUymgaO8EWA5wiSpl0=";
   };
 
+  postPatch = ''
+    # unpin numpy
```

Comment:
let's avoid comments in nix strings since they affect the drv hash

if pythonRelaxDepsHook can't handle this then let's update the comment to explain why

Reply:
> let's avoid comments in nix strings since they affect the drv hash

That would make the diff look worse in case `postPatch` ever has to be extended.